server/client.go

```diff
@@ -2131,6 +2150,15 @@ func (c *client) deliverMsg(sub *subscription, mh, msg []byte) bool {
 
 	client.out.pm++
 
+	// If we are tracking dynamic publish permissions that track reply subjects,
+	// do that accounting here. We only look at c.resps which will be non-nil.
```

Comment:
wrong reference to `c.resps` in this comment?

Reply:
Will fix.